.travis.yml

```diff
@@ -147,7 +147,7 @@ matrix:
     # ========================================================================
     - os: linux
       language: python
-      env: TARGET=lua VARIETY=5.3
+      env: TARGET=lua VARIETY=5.1
```

Comment:
This change will also require changing the line [`App.vue:73`](https://github.com/kaitai-io/kaitai_ci_ui/blob/b6cdedb6a2582601c463251a5351b92c11f2a50e/src/App.vue#L73) in the [`kaitai_ci_ui` repository](https://github.com/kaitai-io/kaitai_ci_ui), otherwise the new `lua 5.1` target won't be shown in the [CI dashboard](https://ci.kaitai.io/).

Reply:
Thanks for hint, i'm actually currently working on Lua matrix, testing the Lua runtime with all compatible versions (5.1 through 5.3), i'll update the kaitai_ci_ui once that is up

Reply:
Should be resolved by https://github.com/kaitai-io/kaitai_ci_ui/pull/8